definitions/variable/technology/electricity-expansion.yaml

```diff
@@ -0,0 +1,27 @@
+# List of variables related to the evolution of the electricity system
+# results of capacity expansion models
+
+- New Capacity|Electricity|{Electricity Input}:
+    description: New capacity for Electricity Input
+      this capacity is added to the existing capacity
+    unit: MWh
+- New Storage|Electricity|Enegy Storage System|{Energy Storage System Type}:
+    description: New storage capacity for Electricity Input
+      this capacity is added to the existing capacity
+    unit: MWh
+- New Capacity|Network|Electricity:
+    description: New capacity for Electricity Input
+      this capacity is added to the existing capacity
+    unit: MWh
+- Removed Capacity|Electricity|{Electricity Input}:
```

Comment:
I suggest to rename this to "Capacity Decommissioning|..."

Reply:
good suggestion